recipes-graphics/mesa/mesa-etnaviv-env_0.1.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Mesa environment variables for etnaviv on xserver"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch
+
+SRC_URI = "file://mesa-etnaviv.conf"
+
+do_configure[noexec] = "1"
+do_compile[noexec] = "1"
+
+do_install() {
+    # MESA global envirronment variables
+
+    # systemd
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+        install -d ${D}${sysconfdir}/systemd/system.conf.d
+        install -m 755 ${WORKDIR}/mesa-etnaviv.conf ${D}${sysconfdir}/systemd/system.conf.d/
+    fi
+
+    # sysvinit - TODO
```

Comment:
It is monster effort in past technology. If this is a prerequisite - sorry

Reply:
We need to keep supporting this as otherwise we'll diverge between the supported init system and we'd like to avoid this.

Reply:
As said: Sorry but I won't go through building an image for sysvinit, take the pain to work with it and find out where to set the knob (am so happy I don't have to use that script crap anymore).
So the story ends here.

Reply:
That's your decision. I am sad to learn you are not interested in contributing to the project as we all contribute to things other people use, and this is how a good community is built.

Your PR was excellent, and I was very interested in getting it in, but if you don't want to be a contributor, that is your call. As said, I am sad to see your reaction to this.

Reply:
Can't leave this uncommented:
Spent several days on this on dunfell. I am happy with the result but dunfell cannot go here because mesa/libdrm/xserver are too old in oe-core. 
I decided 'be a good boy' and contribute upstream - maybe it is useful for others. After doing some adjustments, I built a complete image on master and did some tests - all for being a good boy - zero use case for me.
As result you create a requirement for an init system I call abandonware. For people still using it, adding the exports would be a matter of minutes - for me it occupies one of my machines for hours and further hours to find/test a solution.
With this background your wording
> I am sad to learn you are not interested in contributing to the project as we all contribute to things other people use, and this is how a good community is built.

makes me more than sad.

Reply:
Well, if you end doing the `sysvinit` it enters master and will be ready for you in the future. Especially someone like you, who knows well how those pieces fit together, is a nice addition to our team. So I friendly ask you to spend some more time and finish this.

As said, we all do work for benefit of others. Ultimately, it is a win-win partnership and I am sure we (you and I) already benefit of the work we both did. So I do appreciate your work.